test/types/POD/elliot/formalPOD.chpl

```diff
@@ -0,0 +1,34 @@
+// Check that POD correctly accepts POD types
+
+proc checkPOD(arg: POD) {
+  writeln(arg.type:string, " is POD");
+}
+
+record R {
+  var i: int;
+  var r: real;
+  var b: bool;
+  var t: 2*bool;
+}
+
+var i: int;
+var r: real;
+var b: bool;
+var tup: 2*int;
+var rec: R;
+checkPOD(i);
+checkPOD(r);
+checkPOD(b);
+checkPOD(tup);
+checkPOD(rec);
+
+var Ai: [1..1] int;
+var Ar: [1..1] real;
+var Ab: [1..1] bool;
+var Atup: [1..1] 2*int;
+var Arec: [1..1] R;
+checkPOD(Ai);
+checkPOD(Ar);
+checkPOD(Ab);
+checkPOD(Atup);
+checkPOD(Arec);
```

Comment:
This latter part doesn't seem right. Won't these be not pod and fail to resolve in the test as written?

Reply:
These trigger promotion

Reply:
Oh, right, that being the whole point of the PR... Maybe add a comment in the .chpl for the next clueless chump who reads it ?